types/index.d.ts

```diff
@@ -46,16 +46,15 @@ export interface RenderOptions<V extends Vue, S = {}>
   baseElement?: Element
 }
 
-type ConfigurationArgs = [
```

Comment:
out of curiosity, is this just a syntax change because TS 3.x does not support this type definition syntax?

Reply:
Correct - I believe that format is called "named tuple" which is a new TS v4 feature.